content/developer/reference/backend/views.rst

```diff
@@ -462,6 +462,10 @@ calendar view are:
     (the field to which this values is saved can be controlled through
     ``rec_name``) and tries to create a new event with just that and the clicked
     event time. Falls back to a full form dialog if the quick creation fails
+``quick_create_form_view_id``
+    allows passing the id of a form view to the calendar view. Internally, this is used
+    in the calendar module to show a custom form view when trying to quick create a
+    calendar event
```

Comment:
```suggestion
    view to open when the user tries to quick create a record.
```
Not sure a reference to an example in the code really adds something here (and those kind of examples do not always age well :D)